src/Powercord/migrations/index.js

```diff
@@ -1,5 +1,5 @@
 module.exports = async (powercord) => {
-  const settings = powercord.api.settings.buildCategoryObject('rp-migrations');
+  const settings = powercord.api.settings.buildCategoryObject('coremods/migrations');
```

Comment:
This will not work the first time since the path won't have changed yet when this is running. I think that's fine, just update the migration to delete the existing file and they'll all run again.